controllers/admin/AdminOrdersController.php

```diff
@@ -546,10 +546,10 @@ public function postProcess()
                                     }
                                 }
                             }
-
-                            Tools::redirectAdmin(self::$currentIndex.'&id_order='.(int)$order->id.'&vieworder&token='.$this->token);
```

Comment:
I think this redirect should be kept, but only done if `$this->errors` is empty.

Reply:
So you want to keep the RedirectAdmin, and if nothing is wrong, you want it to reload the order page again?

As far i tested, it seems to work as it should, without the redirect and I cant see a valid reason for it to reload the page if nothing is wrong?